app/grandchallenge/components/utils.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from grandchallenge.components.models import InterfaceKindChoices
+from grandchallenge.core.validators import JSONValidator
+from grandchallenge.hanging_protocols.models import (
+    VIEW_CONTENT_SCHEMA,
+    ViewportNames,
+)
+
+
+def generate_view_content_example(interfaces):
+    images = list(
+        interfaces.filter(kind=InterfaceKindChoices.IMAGE)
+        .order_by("slug")
+        .values_list("slug", flat=True)
+    )
+
+    if not images:
+        return None
```

Comment:
Would it make sense to inform the user that there are no image interfaces and that a viewport needs at least 1 image assigned? You don't need to do that here, and instead check that in the form and add to the helptext accordingly.

Reply:
Done